src/Placeholder.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { mapToCssModules, tagPropType } from './utils';
+import { getColumnClasses } from './Col';
+
+const propTypes = {
```

Comment:
What do you think about just extending `Col.propTypes`? Seems like `Col` props are the main method of specifying `Placeholder` widths.
```
const propTypes = {
  ...Col.propTypes,
  color: PropTypes.string,
  tag: tagPropType,
  animation: PropTypes.oneOf(['glow', 'wave']),
  innerRef: PropTypes.oneOfType([PropTypes.object, PropTypes.func, PropTypes.string]),
  size: PropTypes.oneOf(['lg', 'sm', 'xs']),
};
```